ocs_ci/ocs/ui/pvc_ui.py

```diff
@@ -0,0 +1,95 @@
+import time
+import logging
+from selenium.webdriver.common.by import By
+
+from ocs_ci.ocs.ui.base_ui import BaseUI
+from ocs_ci.ocs.ui.views import pvc
+
+
+logger = logging.getLogger(__name__)
+
+
+class PvcUI(BaseUI):
+    """
```

Comment:
I had provided few comments on UI based create and delete pvc in your previous UI PR. Please check and consider the required comments here too.

Reply:
@hnallurv  I added `take_screenshot` function on BaseUI class